ProcessMaker/Jobs/StartEventConditional.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace ProcessMaker\Jobs;
+
+use ProcessMaker\Models\Process as Definitions;
+
+class StartEventConditional extends BpmnAction
+{
+    public $definitionsId;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct(Definitions $definitions)
+    {
+        $this->definitionsId = $definitions->getKey();
+    }
+
+    /**
+     *
+     */
+    public function action()
```

Comment:
This method doesn't have any logic

Reply:
Yes, that's because this Job only uses the base behavior of BpmnAction to load and evaluate the conditional event